test/lib/fetch.test.ts

```diff
@@ -23,4 +25,16 @@ describe('fetch', () => {
       superagent,
     }, '/homePageContent', '');
   });
+  it('successfully runs fetchPost', () => {
+    Object.defineProperty(superagent, 'post', {
```

Comment:
@kathleenahall we already have a mock for superagent so you don't need to do this here, see the _mocks folder

Reply:
@kathleenahall the test would be an async function and  you would await the function call